server/src/core/api-rest/api-rest-query-builder/factories/find-one-query.factory.ts

```diff
@@ -6,15 +6,13 @@ import { mapFieldMetadataToGraphqlQuery } from 'src/core/api-rest/api-rest-query
 @Injectable()
 export class FindOneQueryFactory {
   create(objectMetadata, depth?: number): string {
+    const objectNameSingular = objectMetadata.objectMetadataItem.nameSingular;
```

Comment:
I think it should always be capitalized!

Reply:
Nope
![image](https://github.com/twentyhq/twenty/assets/29927851/333baae2-4784-4071-9c32-79a91cc5985d)
![image](https://github.com/twentyhq/twenty/assets/29927851/442a808f-02bd-4d6f-bc3b-23a8a34d05b7)

Reply:
nope 
<img width="351" alt="image" src="https://github.com/twentyhq/twenty/assets/29927851/c1804f74-6d95-4601-988d-12f755d4dbd3">